descriptors/ModuleDescriptor-template.json

```diff
@@ -18,8 +18,13 @@
         },
         {
           "methods": [ "POST" ],
-          "pathPattern": "/refresh",
-          "permissionsRequired": []
+          "pathPattern": "/token/sign",
+          "permissionsRequired": [ "auth.signtoken" ]
+        },
+        {
+          "methods": [ "POST" ],
+          "pathPattern": "/token/refresh",
+          "permissionsRequired": [ "auth.signrefreshtoken" ]
```

Comment:
This solution alters

      "id": "authtoken",
      "version": "2.0",

to become authtoken version 2.1.

A different approach simply adds a new

      "id": "authtoken2",
      "version": "1.0",

(or any other id name like "token") and leaves "authtoken" "2.0" as is.

Reply:
Ok added second group. Decided `authtoken2` was a good enough name. Also think it is nice to represent on the top level that there's an "old way" and a "new way" and that there is no overlap.